iis/applications/create.go

```diff
@@ -0,0 +1,23 @@
+package applications
+
+import "fmt"
+
+// Create an IIS Application under the given Website
+func (c *ApplicationsClient) Create(name string, applicationPool string, physicalPath string, parentWebsite string) error {
```

Comment:
could we update the ordering of parameters to match the other resources? e.g.

```suggestion
func (c *ApplicationsClient) Create(name string, parentWebsite string, applicationPool string, physicalPath string) error {
```